src/table/gameopt_settings.ini

```diff
@@ -149,7 +149,8 @@ var      = game_creation.snow_line_height
 type     = SLE_UINT8
 def      = DEF_SNOWLINE_HEIGHT * TILE_HEIGHT
 min      = MIN_SNOWLINE_HEIGHT * TILE_HEIGHT
-max      = MAX_SNOWLINE_HEIGHT * TILE_HEIGHT
+# "max" used to be MAX_SNOWLINE_HEIGHT * TILE_HEIGHT, but this would overflow the storage.
+max      = UINT8_MAX
```

Comment:
Wait a minute here, what unit is snow line height measured in? Because as far as I remember, TILE_HEIGHT is a constant for converting height levels into pixel lengths. Are sub-tile height snow line heights supported even?

Reply:
Don't get me started about this mess. But there is a AfterLoad routine that converts the values here back to their non-`TILE_HEIGHT` counter-part. So in our current code, `snow_line_height` is just a number between 1 and 255. Just for older savegames it can have other values ...

I wasn't sure how to fix this properly btw, so I went with this. "dead" code anyway, was savegame revision 22 .. lol :)

https://github.com/OpenTTD/OpenTTD/blob/master/src/saveload/afterload.cpp#L2774

Reply:
To add to that, I have no clue when clamping is done .. before or after `afterload`. As there is a second `snow_line_height` entry, that is the current version, but the `afterload` suggests older versions can have the value multiplied by `TILE_HEIGHT` too .. so if it is clamped before `afterload`, that line in `afterload` won't work anyway ..

This confuses me :P